src/cgroup.rs

```diff
@@ -588,6 +589,33 @@ pub fn get_cgroups_relative_paths_by_pid(pid: u32) -> Result<HashMap<String, Str
     get_cgroups_relative_paths_by_path(path)
 }
 
+fn get_cgroup_destination(mut mount_root: String, pidpath: String) -> String {
+    if mount_root == "/" {
+        mount_root = String::from("");
+    }
+    pidpath.trim_start_matches(&mount_root).to_string()
+}
+
+pub fn existing_path(paths: HashMap<String, String>) -> Result<HashMap<String, String>> {
```

Comment:
I didn't see where this fn was called?

Reply:
This function is a public function. In my design, this function is optional, it can be called after get_cgroups_relative_paths or get_cgroups_relative_paths_by_pid for coder using this crate.

Reply:
@lifupan